crates/swc/tests/fixture/sourcemap/issue-3854/1-true/output/index.ts

```diff
@@ -4,7 +4,14 @@ Object.defineProperty(exports, "__esModule", {
 });
 var _exportNames = {
     node: true,
-    node: true
+    create: true,
+    trustBox: true,
+    opener: true,
+    stage: true,
+    header: true,
+    source: true,
+    sources: true,
+    seq: true
```

Comment:
it works now how it does in babel - https://babeljs.io/repl#?browsers=Chrome%20%3E=%2030&build=&builtIns=false&corejs=false&spec=false&loose=false&code_lz=JYWwDg9gTgLgBAbzgQQEYGcZQIYGMYBCArsADYAmAplADQoZZ4wDKlAjsWVVHAL5wAzKBBBwA5ADoA9Gkw58nCtXRiA3AChQkWIjgALbOj10Q2MIvLIoAcxNmLV6-j6DhoyVIvUJRGGRUaWtDwSKDY1pQuQiLi0gCipJQglAB2hCRKUGrqmuDBcDAAnmCRCOpwFXAEEOSFNOWV1bXMMOGU9ZVwACpQRJjVAB4dlQBKwLh6XZQDMMMVAPIlKdRzVTWFABKU2NyrLW2rVn64icwQRFC47Q0VR-On55eU6PX80e4AAmBm1FIgNZRSAFctp4EUSroHDY6FCnHQAMJQbYwSheWhwVhsBJJVKzDHsABy0xg2OSaXhEBSADdqCieG83LEpODntl1FIAFQc8ocuAfKhgJG4bAo8hwPqlNESVDrCQpAESUDWAAUAEpeDypOppqC4Lz3kymoUCQD0Gjsjr8rzDHB5VRXDEPEaTVQzRluGzLTpcJTMHBcEiRZQAFxwRHI1Hu6gAHiNaIAfHAALxwNXJxPLADua1qaLVGi98B9KT9WD6hAgA1D4aDaOjPXLgwTydTqvTtso2Yb_UredVBYGuuLfogS2o1cDKLri1S1GbKbTSYznbgM-WUD7A6HvvgmDaE4jdaN-wi89b7azOcKJ8jXGo-e1g_yw_gem23APtajUFj6y2Oznb9EwXNslw7bMjX_bhN0fbcS13R4rk_Kdv2jO4TkoM4LiuNEYXdRx0GjMYJimGYJBdSh42A1MbltU1Q1hQjiMmYlyIBaiAG0AF16lA5ds3Qh5sNvTJlQkcS7WeftYOfHc4HQRDnmQkTuDQ2B7kwxS3TvKBqMXfiUHUjCsKebTROkmTvTk9B2BbMpOj3CJQ2VfTwKvG9MT7NsbRrFCdN_ZpWgiTygNWBThOc1zL0EzThJCnS1W85xfJUmMYpMq54syeNXjgG0Xw0dQTkMZw42_OBiVSchnFkRgFFQo1E3syoAGJHKU9ygsobiW24jROhassewGAB-UNuwrAYWyIFIqAEYBlnIfrWvIYB0GK0BqDGuBmNImA2KoHqUxmuaFsoJbaJa7AjKE0ztvSrTptmyh5sWwqHK69AxPE9r0AY_CbEI48uvjVVQxgPQ1sQWjOghtaJDaz6W1-iRTDAZU0dhaTOk6JEYAuFICkh9BloqDVaKGyblUpwZtthetemG0HweJ6GccqOH0ARmnKxbTGAZVHmBmx9m4DxgmibW0m-ByTpVvW0hsE2qBlXljbkigOmBaY8YWLIijmcl5xmvZzmEbVxXlZbC2lY1kbUbMDH7AFkX2fFqBCc56Xyc6a7jluq4vr9jSMueLWuEcNT_di0zDc5tnRbNq6bpjwO-ediObGVYPjK012cfdz3ie92XKlQd01QY9YE7dyh8Y9mvRYqX6WfhxG2heGHE8ZybW65wae8GVYm5t5W-_Ntb1ZWLv2ZzgOOqTufU-eYfKl4EvyeK9BnAmptysq2aaoYeR0n83fKyay7JL-nbdb2g7uq43quOllrIfIKgUlDXbWIoo64D6qXCo19lTX3-pnJwRE76_3YmDI2jdYbEwRtfFs18Hbo35hA_OuM64Sy9rRH2lR36f2VMQ1I4CKAESgSRGBVA46sxNjjJOZDCYphYegp25gXbSxwfXIuUsCFALgOXLglduiD15ow3hEspGizAUbZBppV7sxYePN-wAP6pGUXwDeOQt7ODXIBHSFUZhVSPnIJg04xy6Ubi1QEOI0ihkMVADiYh7FkhgGIJ-KYwgRDVNKCuFlOjuNxMqEJjiqgC2jM41x4TPFcXoVDWRFQk5xPTlwiBYTEgeOwZUQuRsS60RERQMRziEF5NwQ3ZJOM4lqLido9egjN6K23p1NoWVuAmJRIfegFj8AdJjMDNol8BryMxKSXE1C9b7T_gk5-r83wAU1qGcZ2TJmQXfNQaiJ0XpnQugNH04BsApGAL6FZ7AJlpGjDeCQFIjknMpAASRRCAbinEX6XUORrXAwBsBAjiLNeYAgbznKxGsq5P99YAjefMoRICxkXPBTAWM2spn3wNok42M8FEtRQSmNBaNOGMVyRUfJ-DOiEIqIs7gpDNnLPxGChxyL6YbKWVROB8dqnMLpekxitKllwBGiNABXESVi0qfwkmgjaKHO-A8ksypZXHNOSWUFlzmWopuXcuVKrnlJHZX3cpKSkEtSVfK5wKYzUqq5oSzBlDAZirJcXaVnQvnUB-X89AALyBApvIqkQ3zfn_MBcCrqaqkUoogTrGhUK6GYqNTit1lwg1epDTeFsSaPXBp9aGtoHC7WWAdTwipfCCkurLu6AAMmtGAeqQDKhsmwIkMx1WgubSSJFFJqS0moHAoZEQE35OqZ0DRoYDBGGVGIGUgU2hiDoEna-86TVWt9EutumaU3et9V1Nd_dqW9u0Z0eRjb23qokNdHAhRqYmuvqqQ9RC6WhhPcSM9phCioEoNetu-6oB3uxa6kQOqznyUJC-pF56oCXq_f3FdJY_1NwAyAQNnqt25qciBptYGmUQag0nDdKG01dXg6LRpFL1DNJKm008-9TE9NqifI86wbwjNao2tEVcZ3BXYGiVBK5-2YW49-B8tF4X0UiVGtFtDKIcoYdipObHvxylNMqDi19RXFtJRKstZHaI_v5dwcO9rrABU2HS-N1TgACFTD-tsw6OYmoUzpCQemOI_vU9ijUTcnUCJ04hoDCrYM30YtcrqtzAPKqeS88zcmHOCac4FlTgX3Nea0-SteQj8PZu3W0f1SH3WbsI_ucTRno3TIftFpu8m4uZAkJl1NOa_UcTq6hm8yXRbealb5itoi-1Ma6oO1LsWOCKeKeQB8ZGKOtNZdBGj3Tqq9LqqfTIJmoJbNsVSa6vyInTeoK4jbUAtvxJbGIAQRBSCkGyANUcs56VlOOs9V6513qVH24d5Ur3jkwA46ZpZe3Nufa8TJpJMW24fbSC2MHMANPitLWlsmQjrvrmVIj8cxXC3GecRVxOJqUc8BTAWxwyPrGOsGz59LRTAnfdWzwapQ7_0vf-xEpOkP70VFx2o3HDTdFFRaTVFOocBk8APvN-jTBBdRxDopFjFRcVidWUyyTsbH7vOe8A5TCLGUeMjSVxXMz2JY9Nje00qDTT5oziVknsPnVdYqKN6tmA60NtAy2pFbasMeK7TSWAvbQwPWEgml8cANEtnHXoSdS8AC04Unhzpxbe6HdOEOVA0azuiron3O47dhi92Ar0LtNMR9mpH0uTb59HUOPHhfmMW3WP3TxpdwDatVj8hly-KUF7xgS_P2_N_vEEyoL5ej4GgN9JT6e0dUMhXruhtnsXoCICUFWYqzYgPEmPqShSj3KbX_I4LU_ytA6xZVobUoQGqdNG12uVuyfw4pz1333f_e09J_3RzNXRvjZL3o3nrfJfCTMp0lXgtgxqhHXoHA3rLuPmAY_LCgNKor_rnMJP_IAiJtvhJGJsFtAQbkwkbq6Cbq6GbhkkZpbngtbuTp0CwqPvAZgY_vXofgmswhop_CHkwakIQVjAni_pvt1iUnAtAUfu1lpnZmruPvnq6KnvAYwZoikFzk0kAA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=unambiguous&lineWrap=true&presets=env,react,typescript&prettier=true&targets=Node-10.5&version=7.17.7&externalPlugins=&assumptions=%7B%7D